applications/nrf_desktop/README.rst

```diff
@@ -1860,14 +1860,26 @@ Bootloader
 
 The nRF Desktop application can use one of the following bootloaders:
 
+.. note::
+   See the chosen bootloader subsection below to learn which SoC Series and configurations it supports.
```

Comment:
```suggestion
   See the chosen bootloader subsection to learn which SoC Series and configurations it supports.
```

Reply:
Applied @divipillai suggestion in this matter